rakan/cRakan/src/Runner.h

```diff
@@ -25,41 +27,51 @@ class Runner {
   /*
   * Default constructor.
   */
-  Runner() {}
+  Runner();
```

Comment:
What is this constructor for? Can we ever call it safely? Or should we ever call line 32's constructor?

Reply:
For testing.